apps/rellotge/rellotge.js

```diff
@@ -0,0 +1,131 @@
+// drawWidgets() is added by bootloader.js when loading a clock app, but when you upload via the IDE it just 
+// resets the watch and skips out running bootloader.js completely. So add the relevant code from the bootloader. 
+var WIDGETPOS = {
+  tl: 32,
+  tr: g.getWidth() - 32,
+  bl: 32,
+  br: g.getWidth() - 32
+};
+var WIDGETS = {};
+
+function drawWidgets() {
+  for (var w of WIDGETS) w.draw();
+}
+
+require("Storage").list().filter(a => a[0] == '=').forEach(
+  widget => eval(require("Storage").read(widget)));
+setTimeout(drawWidgets, 100);
```

Comment:
While this is useful for developing, we don't need it for the real app - I notice you've switched to [`Bangle.drawWidgets()`](https://github.com/espruino/BangleApps/blob/d1ab9e2ebf949ee29b78a4e00904417af7c1845b/apps/rellotge/rellotge.js#L117) below which is good, so we can drop these lines here :) (1 through to 17)